Documentation/Contributing/development-flow.md

```diff
@@ -29,10 +29,27 @@ mkdir -p $GOPATH/src/github.com/rook
 cd $GOPATH/src/github.com/rook
 
 # Clone your fork, where <user> is your GitHub account name
-$ git clone https://github.com/<user>/rook.git
+git clone https://github.com/<user>/rook.git
+```
+
+### Add Upstream Remote
+
+First you will need to add the upstream remote to your local git:
+
+```console
+# Add 'upstream' to the list of remotes
 cd rook
+git remote add upstream https://github.com/rook/rook.git
+
+# Fetch all (tags are needed for build process)
+git fetch -a
+
+# Verify the remote was added
```

Comment:
Just a small nit, 
According to me we should verify first and then fetch it after adding a repo.

Reply:
> Just a nit, According to me we should verify first and then fetch it after adding a repo.

I moved the `fetch` operation to the `Building` section.